pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,287 @@ title: What is JSON Schema?
+    data = {
+        "name": "John Doe",
+        "age": 25,
+        "email": "johndoe@example.com"
+    }
+
+    # Load and validate the JSON Schema from step 2
+    with open("schema.json") as schema_file:
+        schema = json.load(schema_file)
+        validate(data, schema)
+    ```
+
+    * We have created an `instance` of the `schema` we defined as an example in step 2.
+    * Since the data is valid, we will not get any output. In case of invalid data, we will get a `jsonschema.exceptions.ValidationError` exception.
+
+4. **Apply to your use case**
+
+    When you have a defined schema and valid JSON data, the possibilities are endless. You can do additional data processing, set up error handling, and integrate your validated data with APIs, other applications, and business logic.
+
+
+## Why developers use JSON Schema
+
+With JSON Schema you can:
+
+* **Describe existing data formats**: JSON Schema allows you to describe the structure, constraints, and data types of your existing JSON data formats. 
+* **Define rules and constraints**: When your JSON documents adhere to these constraints, it becomes easier to exchange structured data between applications because the data follows a consistent pattern.
+* **Clear and readable documentation**: JSON Schema supports the creation of documentation that is easily understandable by both humans and machines.
+* **Highly extensible** and can be tailored to fit your needs.
+    * You can create *custom keywords*, *formats*, and *validation rules* to suit your own requirements.
+* **Validate your data**, which helps you:
+    * **Automate testing**: JSON Schema validation enables automated testing, ensuring that data consistently adheres to the specified rules and constraints.
+    * **Enhance data quality**: By enforcing validation rules, JSON Schema helps ensure the quality of client-submitted data, preventing inconsistencies, errors, and malicious inputs.
+* **Wide range of tools availability**: The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
+
 
-After you define your schema, you validate your JSON documents against the schema. To do this, you can use any supported [validator](https://json-schema.org/implementations.html#validators) in whatever language or context fits your project. There are JSON Schema validation tools for nearly every modern programming language, as well as language-agnostic command-line and browser tools.
+## Why organizations adopt JSON Schema
 
-When you have a defined schema and valid JSON data, the possibilities are endless. You can do additional data processing, set up error handling, and integrate your validated data with APIs, other applications, and business logic.
+* **Simplify testing and validation**: JSON Schema helps to reduce the code's complexity and the development time by simplifying your validation logic.
```

Comment:
We made some changes in the messaging of the Why orgs adopt JSON Schema and this is the last version:

* **Streamline testing and validation**: Simplify your validation logic to reduce your code’s complexity and save time on development. Define constraints for your data structures to catch and prevent errors, inconsistencies, and invalid data.
* **Exchange data seamlessly**: Establish a common language for data exchange, no matter the scale or complexity of your project. Define precise validation rules for your data structures to create shared understanding and increase interoperability across different systems and platforms.
* **Document your data**:Create a clear, standardized representation of your data to improve understanding and collaboration among developers, stakeholders, and collaborators.
* **Vibrant tooling ecosystem**:Adopt JSON Schema with an expansive range of community-driven tools, libraries, and frameworks across many programming languages.